cpp/src/arrow/array/array_list_test.cc

```diff
@@ -434,16 +622,18 @@ class TestListArray : public ::testing::Test {
 
   void TestBulkAppendInvalid() {
     std::vector<int16_t> values = {0, 1, 2, 3, 4, 5, 6};
-    std::vector<int> lengths = {3, 0, 4};
     std::vector<uint8_t> is_valid = {1, 0, 1};
     // Should be {0, 3, 3} given the is_valid array
```

Comment:
Can we fix this comment?

Reply:
Is it OK if I delete it? (I pushed a fixup commit dong that).

Reply:
+1